esy-solve/OpamManifest.re

```diff
@@ -242,12 +242,14 @@ let convertDependencies = manifest => {
 
   let filterOpamFormula = (~build, ~post, ~test, ~doc, ~dev, f) => {
     let f = {
-      let env = var =>
+      let env = var => {
         switch (OpamVariable.Full.to_string(var)) {
-        | "test" => Some(OpamVariable.B(test))
-        | "doc" => Some(OpamVariable.B(doc))
+        | "with-test" => Some(OpamVariable.B(test))
+        | "with-doc" => Some(OpamVariable.B(doc))
```

Comment:
Isn't `test` and `doc` also supported?

Reply:
Those are only available in I believe opam 1.2, should we allow them both?

Reply:
I think so, especially because it seems to have zero downside.

Reply:
Done!